rollup.config.js

```diff
@@ -0,0 +1,51 @@
+import eslint from '@rollup/plugin-eslint'
+import json from '@rollup/plugin-json'
+import replace from '@rollup/plugin-replace'
+import url from '@rollup/plugin-url'
+import svgr from '@svgr/rollup'
+import sass from 'rollup-plugin-scss'
+import typescript from 'rollup-plugin-typescript2'
+
+import { dependencies } from './package.json'
```

Comment:
i would add a top-level comment explaining why we are using rollup at the root of the interface project

Reply:
could be confusing